### ISSUE TITLE
TypeError: Failed to load plugin 'jsx-a11y' declared in '.eslintrc.json'

### ISSUE THREAD

**Issue by vladoyoung** [formaat-design/reshaped#295]
**Describe the bug**
We have the source code of Reshaped set up as our own package and the Github action for `yarn lint` fails. This got fixed by updating the `eslint-plugin-jsx-a11y` to version `6.9.0`

so in my `package.json`
`"eslint-plugin-jsx-a11y": "6.9.0",`

**Expected behavior**
The update to the package, or another fix to come with the source code if possible.

**Screenshots**
![image](https://github.com/user-attachments/assets/0d99fc7a-fee1-4644-97da-8ff12c2e70e9)

**Comment by blvdmitry** [comment]
Interesting. Still works for me locally, but updating to 6.9.0 doesn't break it, so I'm updating it now. I assume you should be fine with just having it updated to 6.9.0 in your source?

**Comment by vladoyoung** [comment]
Yes, that's totally fine!
